lethal-squirrels/Bangazon-Site
# Chaz search improve

## REFERENCE

N/A

## DESCRIPTION
Made the search function better. 
Filtered results by converting the search term to lowercase and comparing it with the product name that has also been converted to lower case. 
Search also returns results if a product name contains the search term at all 

## STEPS TO TEST
1.Search for `C`
1. you should see a list of results that contain the letter C
1. searching for `cool` should return result 'COOL' (if that item is in your database)
